packages/view-types/layer-controller/src/LayerController.js

```diff
@@ -476,6 +477,8 @@ export default function LayerController(props) {
       }
       TransitionProps={{ enter: false }}
       expanded={!disabled && isExpanded}
+      aria-controls="layer-controls options"
```

Comment:
This attribute is supposed to point to the ID of whatever element a given HTML element controls; e.g. with tab panels, tabs get `aria-controls` attributes corresponding to whichever panel they're associated with. 

Looking at what the [MUI docs suggest](https://v4.mui.com/components/accordion/#accessibility), their recommendation is to put `aria-controls` on the `AccordionSummary` to indicate which panel the header controls. The `AccordionSummary` here already has this value set to `layer-${name}-controls` but I couldn't find any matching ID's for that value - maybe that should be provided to the `AccordionDetails`?

Reply:
I read this page a few times, the text is super brief, but the way I understand it is similar: that the `id` of `AccordionDetails` has to be the same as `aria-controls` of `AccordionSummary`. I have now modified the code to do that.